astropy/__init__.py

```diff
@@ -15,9 +15,11 @@
 __minimum_numpy_version__ = '1.10.0'
 
 
-class TooOldPythonError(Exception): pass
+class UnsupportedPythonError(Exception):
+    pass
```

Comment:
This is super-nitpicky, but I think these content-less exceptions are better as a one-liner.

Reply:
Does PEP8 allow one-liners? I really don't like the one liners...

Reply:
I've just checked the same thing. People argue it does: https://github.com/PyCQA/pycodestyle/issues/231#issuecomment-185021750, but the tools doesn't recognize them yet.

Reply:
Off topic: @eteq , just the other day, you cringed when I said one could write an entire C function in one line...

p.s. FWIW two lines here looks cleaner to me.